paimon-trino-common/src/main/java/org/apache/paimon/trino/TrinoPageSourceBase.java

```diff
@@ -176,11 +178,17 @@ private void appendTo(Type type, DataType logicalType, Object value, BlockBuilde
                 BigDecimal decimal = ((Decimal) value).toBigDecimal();
                 type.writeLong(output, encodeShortScaledValue(decimal, decimalType.getScale()));
             } else if (type.equals(TIMESTAMP_MILLIS)) {
+                // TimestampType mapped to ShortTimestampType of TIMESTAMP_MILLIS
                 type.writeLong(
                         output,
                         ((Timestamp) value).getMillisecond() * MICROSECONDS_PER_MILLISECOND);
-            } else if (type.equals(TIME_MICROS)) {
-                type.writeLong(output, (int) value * MICROSECONDS_PER_MILLISECOND);
```

Comment:
Why replacing `TIME_MICROS` with `TIME_MILLIS` instead of simply adding it?

Reply:
https://github.com/choucmei/incubator-paimon-trino/blob/120e693d24374f42b25dabb0308bfa187d1ae6dd/paimon-trino-common/src/main/java/org/apache/paimon/trino/TrinoTypeUtils.java#L150
Because TIME_MILLIS can't be mapped.

Reply:
Because TIME_MILLIS can't be mapped. 
https://github.com/apache/incubator-paimon-trino/blob/45e003a1180fd959f59d11037677e742139f15fe/paimon-trino-common/src/main/java/org/apache/paimon/trino/TrinoTypeUtils.java#L150